tensorflow/c/experimental/filesystem/modular_filesystem.cc

```diff
@@ -390,6 +390,39 @@ void ModularFileSystem::FlushCaches(TransactionToken* token) {
   if (ops_->flush_caches != nullptr) ops_->flush_caches(filesystem_.get());
 }
 
+Status ModularFileSystem::SetConfiguration(const std::string& name,
+                                           const std::vector<string>& values) {
```

Comment:
We need multiple overloads to set int/floating values too.

Reply:
Thanks @mihaimaruseac . Added additional functions for int and real types.